### PR TITLE
[EC-482] Additional fields in syncing api (live after death)

### PR DESCRIPTION
**Pull request by KonradStaniec**
## Description

Adding more info to `eth_syncing`.

- original target: `phase/release1_1`

### CODE REVIEW

File: src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala [hunk 1]
```diff
@@ -325,6 +325,10 @@ class FastSync(
         .addPendingNodes(hashesToRequest.flatten)
         .copy(downloadedNodesCount = syncState.downloadedNodesCount + nodeData.values.size)
 
+
+      appStateStorage.putPulledStateNodes(syncState.downloadedNodesCount)
```
Comment (LukasGasior1) [inline src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala]: I wonder, do we need to save those to leveldb? We already do that as a part of sync state (persistSyncState). Maybe we can load it from there on startup and keep as a var somewhere? Or maybe EthService could just ask FastSync for `SyncState`?

Comment (KonradStaniec) [reply src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala]: good call, it is just case of using existing `FastSyncStorage`  to get currently persisted state.

Comment (LukasGasior1) [reply src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala]: this should work too although keep in mind this state is only persisted every 1 minute or so (I think)

Comment (KonradStaniec) [reply src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala]: yep every minute, but I think we will be good with it, we just want to give the user some feedback that something is happening

**Comment by mirkoAlic** [comment]
After a little discussion with @ntallar, we conclude is better to not add BREAKS_API changes like this one, given some tools could be parsing the `eth_syncing` response.

**Pull request closed without merging**